edx_rbac/utils.py

```diff
@@ -76,27 +77,33 @@ def request_user_has_implicit_access_via_jwt(decoded_jwt, role_name):
     """
     jwt_roles_claim = decoded_jwt.get('roles', [])
 
-    feature_roles = []
+    feature_roles = {}
     for role_data in jwt_roles_claim:
-        role_in_jwt = role_data.split(':')[0]  # split should be more robust because of our cousekeys having colons
+        # split should be more robust because of our cousekeys having colons
+        role_in_jwt, context_in_jwt = role_data.split(':')
```

Comment:
I think we can use [str.partition](https://docs.python.org/2/library/stdtypes.html#str.partition) instead of `split` if we expect `role_data` to have only one `:` colon 
```python
role_in_jwt, separator, enterprise_id_in_jwt = role_data.partition(':')
```

Reply:
There's a builtin for everything. This is definitely a better choice 👍 

I don't think we had planned on have roles claims with more than 1 section, so I think str.partition is the best thing (at least for now)

Reply:
One consideration for the future would be course keys, which DO have `:` in them... we would need to find a proper way to escape them. For our immediate use-case though, I think this solution is fine